CHANGELOG.next.toml

```diff
@@ -197,6 +197,12 @@ references = ["smithy-rs#1935"]
 meta = { "breaking" = true, "tada" = false, "bug" = false }
 author = "jdisanti"
 
+[[aws-sdk-rust]]
```

Comment:
Would be good to add a broader `[[smithy-rs]]` entry for the overall required query param handling change as well.